app/components/friendship-page/template.hbs

```diff
@@ -42,4 +42,17 @@
     transactions=sortedUserTransactions
     transactionUpdated="updateBalance"
   }}
+
+  <button class="btn settle-transaction" {{action 'wantsSettled'}} style="width:100%">
```

Comment:
To support other full-width buttons. I recommend adding a class under this: https://github.com/tbbr/tbbr-client/blob/master/app/styles/base.scss#L127

Something like this 

```
.btn---full-width {
    width: 100%
}
```

and then you can just have 
`<button class="btn btn--full-width" {{action 'wantsSettled'}}>`